space2stats_api/src/app/utils/db_utils.py

```diff
@@ -10,6 +11,9 @@
 DB_PASSWORD = settings.DB_PASSWORD
 DB_TABLE_NAME = settings.DB_TABLE_NAME or "space2stats"
 
+conninfo = f"host={DB_HOST} port={DB_PORT} dbname={DB_NAME} user={DB_USER} password={DB_PASSWORD}"
```

Comment:
An arguably cleaner implementation would be to add a `db_connection_string`  property to `Settings` so you could skip lines 7-14.

Reply:
Much cleaner! Just integrated.